repository/repository/targets.json

```diff
@@ -62,34 +81,82 @@
 					],
 					"terminating": false,
 					"threshold": 1
+				},
+				{
+					"keyids": [
+						"9e7d813e8e16062e60a4540346aa8e7c7782afb7098af0b944ea80a4033a176f"
+					],
+					"name": "revocation",
+					"paths": [
+						"*"
```

Comment:
This should be limited to revocation.list, correct?

Reply:
What is the risk associated with this? A single online key can sign for any targets?

Reply:
Yes, I don't know how that interacts with overriding existing targets. I have to try/read if the spec says anything.

Reply:
Top-level targets will be prioritized over further delegations, so the key can't overwrite keys, but it could add targets.

Reply:
Do we have anywhere in client code that is searching by name with a regex anymore? If it's just by custom status, then I think we're ok. If not, then we should consider regenerating.
If so, then for v5, we should limit both this and staging to not use a wildcard.

Reply:
> If it's just by custom status, then I think we're ok.

Why? Delegations can include custom status.

Reply:
Ah, good point, I was thinking that it'd just be signing off on targets without custom metadata. This seems like an issue then, since the staging or revocation keys can add targets like a new CA root or different log key.

Reply:
yeah, i'll rewind and re-sched. closing this pr

Reply:
actually hayden, i'm not sure that changes anything.

Reply:
path names don't matter anymore, just custom metadata does.

i think the fix actually belongs in sigstore TUF client. 

e.g. what if the online key modifies revocation.list to be a root certificate and adds custom metadata?

Reply:
we actually want delegations to be delegated trust to certain USAGES in the custom metadata. not to certain PATHS.

Reply:
Should delegations not be able to use custom metadata? Or maybe a lighter version of that would be not allowing them to use the custom metadata we define (fulcio/rekor/ctfe/etc)?

Reply:
i think we might need to move this to v5 -- i'm not sure how to handle it without investigating where else i can put custom metadata into TUF besides on target files.

ideally we have custom metadata on each ROLE of the delegation deisgnating, for e.g. that they are allowed to use REKOR usage (e.g. for rekor delegation), or nothing at all maybe.

Reply:
Discussed offline, this will be resolved in sigstore's TUF client for now by disallowing delegations to specify custom metadata.